locale/de/translation.json

```diff
@@ -1,90 +1,90 @@
+    "Deleted by creator on ": "Vom Verfasser gelöscht am ",
+    "Deleted by moderation on ": "Von der Moderation gelöscht am ",
+    "Read Less": "Zeige weniger",
+    "Read More": "Zeige mehr",
+    "{{ show_hide }} {{ comments }} answers": "{{ show_hide }} {{ comments }} Antwort",
+    "{{ show_hide }} {{ comments }} answers_plural": "{{ show_hide }} {{ comments }} Antworten",
+    "Show": "Zeige",
+    "Hide": "Reduziere",
+    "up-votes": "Stimmen dafür",
+    "click to up vote": "Klicke, um dafür zu stimmen",
+    "down-votes": "Stimmen dagegen",
+    "click to down vote": "Klicke, um dagegen zu stimmen",
+    "Reply": "Antworte",
+    "Share": "Teile",
+    "Comment must be no longer than 4000 characters.": "Kommentar darf nicht länger als 4000 Zeichen sein.",
+    "Comment input": "Vorschlag kommentieren",
+    "Type your comment here": "Tippe dein Kommentar hier ein",
+    "Enter your comment": "Trage dein Kommentar ein",
+    "{{ count }} Entries": "{{ count }} Einreichung",
+    "{{ count }} Entries_plural": "{{ count }} Einreichungen",
+    "by": "von",
+    "profile": "Benutzerkonto",
+    "click to go to profile and settings": "Klicke, um zu Benutzerkonto und Einstellungen zu gelangen",
+    "Explore": "Entdecke",
+    "Recently Added": "Vor Kurzem hinzugefügt",
+    "This idea will be deleted.": "Diese Idee wird gelöscht.",
+    "Your idea was deleted.": "Deine Idee wurde gelöscht.",
+    "Ok": "Ok",
+    "Reference No.:": "Reference No.:",
+    "n/a": "k.A.",
+    "Idea title is Required": "Ideentitel ist erforderlich",
+    "Description is required": "Beschreibung ist erforderlich",
+    "Please confirm the copyright": "Bitte bestätige die Urheberrechte",
+    "An error occured": "Ein Fehler ist aufgetretten",
+    "Your idea was added.": "Deine Idee wurde hinzugefügt.",
+    "Your idea was updated.": "Deine Idee wurde aktualisiert.",
+    "Please try again.\n": "Bitte versuchen Sie noch einmal.\n",
+    "Required fields are missing.": "Pflichtangaben fehlen.",
+    "Try again.": "Versuche noch einmal.",
+    "Submit a new idea for this project": "Reiche eine neue Idee für dieses Projekt ein",
+    "Idea title": "Ideentitel",
+    "Enter your idea title": "Trage dein Ideentitel ein",
+    "next": "weiter",
+    "Enter your idea description": "Trage deine Ideenbeschreibung ein",
+    "Idea Category": "Ideenkategorie",
+    "Idea Labels": "Ideenmerkmale",
+    "Add Image": "Füge ein Foto hinzu",
+    "Description must be no longer then 10000 characters": "Die Beschreibung darf nicht länger als 10000 Zeichen sein",
+    "Add description": "Beschreibung hinzufügen",
+    "Follow": "Folgen",
+    "Contact for Questions": "Kontakt für Fragen",
+    "Telephone:": "Telefon:",
+    "Email:": "E-Mail:",
+    "Website:": "Webseite:",
+    "No ideas found. Be the first to add an idea!": "Noch keine Idee vorhanden. Seien Sie mit ihrer Idee die/der erste!",
+    "It starts on {{startDate}}.": "Es beginnt am {{startDate}}.",
```

Comment:
@SandisStudi In the app it says now 'Beteiligung ist im Moment nicht möglich. Es beginnt am {{startDate}}'. Maybe 'Sie beginnt ..' would be better, right? same for the following 2 lines..